env0/resource_environment.go

```diff
@@ -185,10 +185,10 @@ func setEnvironmentSchema(d *schema.ResourceData, environment client.Environment
 	d.Set("id", environment.Id)
 	d.Set("name", environment.Name)
 	d.Set("project_id", environment.ProjectId)
-	d.Set("workspace", environment.WorkspaceName)
-	d.Set("auto_deploy_by_custom_glob", environment.AutoDeployByCustomGlob)
-	d.Set("ttl", environment.LifespanEndAt)
-	d.Set("terragrunt_working_directory", environment.TerragruntWorkingDirectory)
+	safeSet(d, "workspace", environment.WorkspaceName)
```

Comment:
Minor bug. These calls return errors (fields are not in the schema).
With TF_ACC it panics.

safeSet first checks if the field is in the schema.